admin_guide/upgrading_citus.rst

```diff
@@ -63,14 +63,17 @@ Step 1. Update Citus Package
 Step 2. Apply Update in DB
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After installing the new package, run the extension upgrade script:
+After installing the new package and restarting the database, run the extension upgrade script.
 
 .. code-block:: bash
 
+  # you must restart PostgreSQL before running this
   psql -c 'ALTER EXTENSION citus UPDATE;'
 
-  psql -c '\dx'
   # you should see the newer Citus version in the list
+  psql -c '\dx'
+
+During a major version upgrade Citus will refuse to run distributed queries until ALTER EXTENSION is executed.
```

Comment:
I tried having this as a `.. note::` but it's close to another note and looked too busy. Could go back to that though if you think this point merits the extra attention.